Readme.md

```diff
@@ -0,0 +1,3 @@
+Adds the Pipeline step ‘stage’ to delineate portions of a build.A
+component of [Pipeline
+Plugin](https://wiki.jenkins.io/display/JENKINS/Pipeline+Plugin).
```

Comment:
I would rename this file to `README.md` and replace the current content with this: 

```
# Pipeline: Stage Step Plugin

[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-stage-step)](https://plugins.jenkins.io/pipeline-stage-step)
[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/pipeline-stage-step-plugin?label=changelog)](https://github.com/jenkinsci/pipeline-stage-step-plugin/blob/master/CHANGELOG.md)
[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-stage-step?color=blue)](https://plugins.jenkins.io/pipeline-stage-step)

## Introduction

Adds the Pipeline `stage` step, which delineates portions of a build.

## Version History

See [the changelog](CHANGELOG.md).
```

Reply:
Sounds Cool :+1: